apis/Google.Cloud.Spanner.Data/docs/connection_string.md

```diff
@@ -64,4 +64,49 @@ v1.0.
 
 Example:
 
-- UseClrDefaultForNull=true
\ No newline at end of file
+- UseClrDefaultForNull=true
+
+# Timeout
+
+The default timeout used for `SpannerCommand.CommandTimeout`,
+`SpannerTransaction.CommitTimeout`, and other Spanner network
+operations. Defaults to 60 seconds.
+
+Example:
+
+- Timeout=30
+
+# AllowImmediateTimeouts
+
+When set to False, a timeout of 0 means an indefinite timeout.
+When set to True, a timeout of 0 means an immediate timeout.
+Defaults to False.
+
+Example:
+
+- AllowImmediateTimeouts=true
+
+# MaximumGrpcChannels
+
+The maximum number of gRPC channels to use per connection using the
+same settings. Defaults to 4. This setting rarely needs to be
+modified.
+
+Example:
+
+- MaximumGrpcChannels=8
+
+# MaxConcurrentStreamsLowWatermark
+
+The maximum number of concurrent streams per gRPC channel before
+using a new channel. Defaults to 20. This setting rarely needs to be
+modified.
```

Comment:
nit: no example here, fine with it, just broke the pattern slightly :)

Reply:
Fixed, thanks. (Admittedly I'd be shocked if this were ever needed. It's really just for completeness.)